main/src/com/google/refine/operations/recon/ReconOperation.java

```diff
@@ -298,6 +298,8 @@ public void run() {
                             // TODO add EvalError instead? That is not so convenient
                             // for users because they would lose the cell contents.
                             // Better leave the cell unreconciled so they can be reconciled again later.
+                            Cell oldCell = entry.cell;
```

Comment:
This variable declaration does not do anything and can be removed

Reply:
I think it is being used in the next line,should we use entry.cell directly?

Reply:
It looks like it's not there anymore, great.